packages/documentation-framework/templates/mdx.js

```diff
@@ -84,6 +85,12 @@ const MDXChildTemplate = ({
           {' '}To learn more about the process, visit our <Link to="/get-started/about#major-release-cadence">about page</Link>.
         </InlineAlert>
       )}
+       {(template || source === 'react-template') && (
+        <InlineAlert title="Templates" variant="info">
```

Comment:
@edonehoo Can you please review the content here?

Reply:
- can we transform the component name variable to be lowercase?